source/math/FloatExtension.ooc

```diff
@@ -120,4 +120,10 @@ extend Float {
 		result := coefficient toString() >> "E" & power toString()
 		result
 	}
+	getTranslationFromRotation: static func (rotation: This, zDistance: This) -> This {
```

Comment:
`getTranslationFromRotation: static func (rotation, zDistance: This) -> This {` didn't work?